docs/book/driver-deployment/installation.md

```diff
@@ -178,33 +177,20 @@ For security purposes, it is advised to remove this configuration file.
 rm csi-vsphere.conf
 ```
 
-## Create Roles, ServiceAccount and ClusterRoleBinding for vSphere CSI Driver <a id="csi_service_account"></a>
-
-Create ClusterRole, ServiceAccounts and ClusterRoleBinding needed for installation of vSphere CSI Driver
-
-```bash
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/manifests/v2.0.0/vsphere-7.0/vanilla/rbac/vsphere-csi-controller-rbac.yaml
-```
-
-Click [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.0.0/vsphere-7.0/vanilla/rbac) to view the roles assigned to vSphere CSI Driver.
-
 ## Install vSphere CSI driver <a id="install"></a>
 
-Our CSI Controller runs as a Kubernetes deployment, with a replica count of 1. For version `v2.0.0`, the `vsphere-csi-controller` Pod consists of 6 containers – the CSI controller, External Provisioner, External Attacher, External Resizer, Liveness probe and [vSphere Syncer](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/pkg/syncer).
+Before you deploy the vSphere CSI driver, refer to the [Compatibility](../compatiblity_matrix.md) page to view the supported kubernetes versions for a particular vSphere CSI version and [feature support](../supported_features_matrix.md) page to see what all features are available on that version.
 
-```bash
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/manifests/v2.0.0/vsphere-7.0/vanilla/deploy/vsphere-csi-controller-deployment.yaml
-```
+- Choose the manifests pertaining to the latest version of vSphere CSI driver from the [Github](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/) repository. For example, if you want to deploy vSphere CSI v2.1.1 on a vSphere 7.0u1 environment, you will choose [this](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.1.1/vsphere-7.0u1/vanilla) folder. 
```

Comment:
Do we prefer customers using manifest YAMLs from v2.1.1 tag or is this link from master branch okay?

Reply:
I thought we're removing the vsphere versions from v2.2 onwards? So should we suggest what the latest repository structure looks like?

Reply:
Yes this PR will be modified again after v2.2 release is out. I raised this PR right now to fix the broken links we have in our installation page.